posts/2022-02-24-Rust-1.59.0.md

```diff
@@ -200,6 +200,26 @@ The following previously stable functions are now `const`.
 - [`mem::MaybeUninit::assume_init_ref`][muninit_init_ref]
 - [`ffi::CStr::from_bytes_with_nul_unchecked`][cstr_from_bytes]
 
+### Incremental compilation off by default
+
+The 1.59.0 release disables incremental by default (unless explicitly asked for
+by an environment variable, with `RUSTC_FORCE_INCREMENTAL=1`). This mitigates a
+known bug which can cause deserialization errors (and panics) during compilation
+with incremental compilation turned on.
+
+This particular bug is preexisting, and was exposed due to other bugs fixed in
+incremental in the 1.59 release cycle. As we continue to fix bugs in incremental
+compilation, new bugs may get surfaced which were previously hidden; due to a
+reluctance to ship unknown problems we have avoided aggressively backporting
+incremental-related changes. This is in part due to having limited ability to
+exercise incremental in automated tests, which means that we currently must rely
+on user-submitted issues to determine the likely level of impact and make
+decisions to enable or disable incremental for a given release cycle.
+
+This particular bug is already fixed on beta and should roll out to stable 1.60.
+We have not made a decision yet as to whether 1.60 will re-enable incremental
+compilation by default.
```

Comment:
I think this paragraph should go second, or at least the first sentence should be moved up. The wording "we have avoided aggressively backporting incremental-related changes" above makes more sense given this context. You could potentially pull that into this paragraph as well, which frames the situation more positively – we already have a fix, but want to ensure that we're shipping high quality releases by following the release process. This feature only affects build times so it's safest to disable for this release.

Also, "This particular bug is" is repeated at the beginning of both paragraphs.